tests/test_validation.py

```diff
@@ -848,11 +848,11 @@ def validate_hypotest(pdf, data, mu_test, expected_result, tolerance=1e-6):
     params=[
         ('setup_1bin_shapesys', 1e-6),
         ('setup_1bin_lumi', 4e-6),
-        ('setup_1bin_normsys', 2e-9),
+        ('setup_1bin_normsys', 6e-6),
         ('setup_2bin_histosys', 8e-5),
-        ('setup_2bin_2channel', 1e-6),
+        ('setup_2bin_2channel', 2e-5),
         ('setup_2bin_2channel_couplednorm', 1e-6),
-        ('setup_2bin_2channel_coupledhistosys', 1e-6),
+        ('setup_2bin_2channel_coupledhistosys', 5e-5),
```

Comment:
this shouldn't get changed, should it?

Reply:
Yeah that's a good point. As these tolerances are coming into play with

```
assert abs(CLs_obs - expected_result['obs']) / expected_result['obs'] < tolerance
```

the expected values should be updated if anything.